packages/terra-application-utility/docs/README.md

```diff
@@ -11,9 +11,26 @@
 
 ```jsx
 import React from 'react';
-import ApplicationUtility from 'terra-application-utility';
+import { ApplicationHeaderUtility, ApplicationMenuUtility, UserData } from 'terra-application-utility';
+
+<ApplicationHeaderUtility
```

Comment:
It was mentioned these components are expected to correlate to the header and menu layouts. We should update these examples to reflect a true implementation.